feat(api): Configurable response time badge thresholds (#267)
TwiN/gatus: README.md

| `endpoints[].ui`                                | UI configuration at the endpoint level.                                                                                                     | `{}`                       |
| `endpoints[].ui.hide-hostname`                  | Whether to include the hostname in the result.                                                                                              | `false`                    |
| `endpoints[].ui.dont-resolve-failed-conditions` | Whether to resolve failed conditions for the UI.                                                                                            | `false`                    |

> | `endpoints[].ui.badge.reponse-time`  | List of response time thresholds. Each time a threshold is reached, the badge has a different color.                                        | `[50, 200, 300, 500, 750]` |

Reviewer: Could you fix the alignment? 🙏

> | `endpoints[].ui.badge.reponse-timer.thresholds` | Threshold array for response time badge color grading (values have to raise in an ascending order)                                          | `[50, 200, 300, 500, 750]` |

Reviewer: ```suggestion
| `endpoints[].ui.badge.reponse-time.thresholds`  | List of response time thresholds. Each time a threshold is reached, the badge has a different color.                                        | `[50, 200, 300, 500, 750]` |
```

> | `endpoints[].ui.badge.reponse-time.thresholds`  | List of response time thresholds. Each time a threshold is reached, the badge has a different color.                                        | `[50, 200, 300, 500, 750]` |

Reviewer: ```suggestion
| `endpoints[].ui.badge.response-time.thresholds` | List of response time thresholds. Each time a threshold is reached, the badge has a different color.                                        | `[50, 200, 300, 500, 750]` |
```

| `ui.header`                                     | Header at the top of the dashboard.                                                                                                                | `Health Status`            |
| `ui.logo`                                       | URL to the logo to display.                                                                                                                        | `""`                       |
| `ui.link`                                       | Link to open when the logo is clicked.                                                                                                             | `""`                       |
| `ui.buttons`                                    | List of buttons to display below the header.                                                                                                       | `[]`                       |
| `ui.buttons[].name`                             | Text to display on the button.                                                                                                                     | Required `""`              |

> | `ui.buttons[].link`                             | Link to open when the button is clicked.                                                                                                           | Required `""`              |

Reviewer: Why is this being removed? the root-level `ui` configuration is different from the endpoint-level `ui` configuration.

Also, I'm sorry for making you wait for so long - thank you for your patience :)

Author: idk, wasn't aware of this removal at all. We rushed it via the online diff editor. Maybe this is sourced in another branch/merge and my branch isn't on yet. This feature is completly new to me.
Will readd it asap

| `maintenance`                                   | [Maintenance configuration](#maintenance).                                                                                                  | `{}`                       |

> 

Reviewer: ```suggestion
```
